lib/sign.js

```diff
@@ -18,11 +18,11 @@ import {
 import {name} from './name.js';
 import {requiredAlgorithm} from './requiredAlgorithm.js';
```

Comment:
We should use `requiredAlgorithm` as the param name in the constructor below, so mapping this import to `_requiredAlgorithm`.

```suggestion
import {requiredAlgorithm as _requiredAlgorithm} from './requiredAlgorithm.js';
```

Notably, using anything other than a P-256 ECDSA key is currently non-standard usage, but I think it's ok to allow this implementation to support it.